docs/how-to/hip_runtime_api/memory_management/device_memory.rst

```diff
@@ -0,0 +1,52 @@
+.. meta::
+  :description: This chapter describes the device memory of the HIP ecosystem
+                ROCm software.
+  :keywords: AMD, ROCm, HIP, device memory
+
+.. _device_memory:
+
+*******************************************************************************
+Device memory
+*******************************************************************************
+
+Device memory exists on the device, e.g. on GPUs in the video random access
+memory (VRAM), and is accessible by the kernels operating on the device. Recent
+architectures use graphics double data rate (GDDR) synchronous dynamic
+random-access memory (SDRAM) such as GDDR6, or high-bandwidth memory (HBM) such
```

Comment:
Specific information like this does not belong in the documentation and becomes outdated very quickly - E.g. the MI300 is backed by HBM3e.

You could however mention, as a general concept, that device memory has ~an order of magnitude higher bandwidth when accessed by kernels when compared to the cpu accessing normal RAM.

Reply:
Added to separate issue.